### PR TITLE
Show an error if the alternate title field is empty

### PR DESCRIPTION
**Pull request by vimleshtna**
Show an error if the alternate title field is empty only when user is selected "yes" for the TitleClosed field

### CODE REVIEW

File: app/controllers/util/FormField.scala [hunk 1]
```diff
@@ -164,7 +164,7 @@ object RadioButtonGroupField {
     def dependenciesError: Option[String] = radioButtonGroupField.dependencies
       .get(option)
       .flatMap(_.flatMap {
-        case textField: TextField         => TextField.validate(dependencies(textField.fieldId), textField)
+        case textField: TextField         => TextField.validate(dependencies(textField.fieldId), textField.copy(isRequired = true))
```
Comment (TomJKing) [inline app/controllers/util/FormField.scala:167]: Does this impact other text fields that are not required and left empty?

**Review by TomJKing — COMMENTED** [review]
Looks good. One minor comment.

File: app/controllers/util/FormField.scala (hunk 1 above)
Comment (vimleshtna) [reply app/controllers/util/FormField.scala:167]: I don't think so as it's only required for the radio button field. In this case, if the user selects "yes" then a dependent field is required. For the other TextField, we are directly calling the `TextField.validate` method.

**Review by TomJKing — APPROVED** [review]
Looks good. Thanks

**Pull request merged**